### DISCUSSION TITLE
Add t3 tmogs to the mythics reward system.

### DISCUSSION THREAD

**Discussion by joesill** [chromiecraft/chromiecraft#5868] [Suggestions]
As per the title.

**Comment by Aethereus-ManaOnion** [comment]
I think it could be seen as unfair for the Hardcore players who have received their t3's as a reward for their heroic achievements

**Reply by balleny** [reply]
at the same time no one forces you to play either mythic modes or hc challenge.
in the end it would be just one of two ways to get access to t3 tmog. similar to the approach of wotlk glyphs with currently can be accessed via honor or gold

**Comment by Amrx2** [comment]
Hi joesill, this isn't something we have in mind for mythics. If you haven't already there is a survey out where some options were covered there - https://forms.gle/E8r56YFBzJ1GzTmA9

Cheers,
Construct
